index.html

```diff
@@ -151,7 +151,7 @@ <h2>Project name goes here</h2>
         <div class="container-3">
 
           <div class="project-container project-6">
-            <img class="project-image" src="images/project-6.jpg" alt="project-6 image">
+            <img class="project-image" src="images/project-6.jpg" alt="macbook pro 16 with smartphone">
 
             <div class="single-project-info">
               <h2 class="project-name">Project name goes here</h2>
```

Comment:
- [ ] Please change this to `h3` instead of `h2` as the Heading levels should have a meaningful hierarchy.